docs/configuration.md

```diff
@@ -1957,7 +1957,7 @@ The TCP address to connect to, including a port number.
       username: [username]
       password: [password]
 
-Proxy enables a registry to be configured as a pull through cache to the official Docker Hub.  See [mirror.md](mirror.md) for more information
+Proxy enables a registry to be configured as a pull through cache to the official Docker Hub.  See [mirror.md](mirror.md) for more information. Note that disables local pushes.
```

Comment:
We will want this feature in the longer term.  I think this could be more accurately stated as 'Pushing to a registry configured as a pull through cache is currently unsupported'.